### PR TITLE
Bug/search functionality

### PR DESCRIPTION
**Pull request by kevinhartmann23**
@JeffKersting

## *Purpose:*
*What does this merge do? Check all that apply.*
- [ ] Adds a new feature
- [ ] Adds testing for new feature
- [x] Improves an existing feature
- [x] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
Search results were broken from error handling, changed useEffect to open rather than checking term length. 

## *What are the relevant tickets?*
Closes _

## *Bugs:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [ ] I have performed a self-review of my own code
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

**Comment by kevinhartmann23** [comment]
@JeffKersting if you could take a look at this PR it would be greatly appreciated. Thank you!!

File: src/components/App/App.js [hunk 1]
```diff
@@ -81,24 +82,25 @@ const App = () => {
...
-            <Route path='/account' component={AccountInfo} />
-            <Route path='/resume' component={Resume} />
-          </Switch>
-        </ThemeProvider>
+      <GlobalStyles />
+      <ThemeProvider theme={theme}>
+        <Header />
+        <Sidebar />
+        <Switch>
+          <Route path='/' exact component={Dashboard} />
```
Comment (JeffKersting) [inline src/components/App/App.js:90]: App return statement is really clean and easy to follow.

File: src/components/SearchForm/SearchForm.js [hunk 2]
```diff
@@ -48,7 +48,11 @@ const SearchForm = () => {
       resetSearchData()
       setSearch('')
     }
-  }, [termData.length, resetSearchData])
+  }, [])
+
+  useEffect(() => {
```
Comment (JeffKersting) [inline src/components/SearchForm/SearchForm.js:53]: In testing, can see how this is affected by a page refresh.

File: src/components/SearchForm/SearchForm.js [hunk 3]
```diff
@@ -101,6 +105,13 @@ const SearchForm = () => {
               </Panel>
             </div>
         }
+        {error &&
+          <div className='loading-container'>
+            <Panel variant='well' style={{ width: '100%', padding: '1rem', display: 'flex', flexDirection: 'column', justifyContent: 'space-between', alignItems: 'center' }}>
```
Comment (JeffKersting) [inline src/components/SearchForm/SearchForm.js:110]: Making some of the inline styling multiline could look cleaner. Or creating a utility file that contains longer inline styling as exports to be applied in component return statements.

File: src/contexts/AppContext.js [hunk 4]
```diff
@@ -24,8 +25,13 @@ export default function AppProvider({ children }) {
     try {
       const result = await apiCalls.requestTermsInfo(terms)
       const data = await result.list
-      const sortedData = await sortIncomingData(data)
-      await setTermData(sortedData)
+      if(!data.length){
```
Comment (JeffKersting) [inline src/contexts/AppContext.js:28]: Easy to follow the logic in the async functions.

**Review by JeffKersting — COMMENTED** [review]
Everything looks good! I am unfamiliar with the best practices of inline styling, but could be made multiline for easier readability, or could import the inline styling from a utility file to cleanup the return statements a little. This is a minor point, beyond that everything looks great and easy to follow!

**Pull request merged**